### PR TITLE
docs: update README.md

### PR DESCRIPTION
**Pull request by MENGZHEGENG**
Update README.md on instructions about outputting the extracted alignments in text_grids.

### CODE REVIEW

**Review by joanise — CHANGES_REQUESTED** [review]
This is a good idea, but I think I would add more details. To review this, I tried to figure out what `preprocessing.save_dir` is referring to, and it's not obvious to me. My `config/everyvoice-aligner.yaml` file does not have that parameter. If it has a default value, I would explicitly state it here, e.g.,

    The extracted alignment can be found in `preprocessing.save_dir`, which is `foo/bar` by default.

Also, if `utt_num` is a count rather than an index, I would use `utt_count` or `n_utt`, because `utt_num` brings my brain to "the index number of the utterance you want" rather than "how many utterances you have". `utt_count` would avoid any such ambiguity.

While reviewing this, I noticed that `dfaligner -h` and its subcommand `dfaligner cmd -h` messages need some serious work! But that's for another PR... Maybe I'll make an issue.

**Comment by MENGZHEGENG** [comment]
Thanks Eric for the insightful review! The changes are made accordingly.

**Review by joanise — APPROVED** [review]
Looks good, thanks!

File: README.md [hunk 1]
```diff
@@ -82,6 +82,10 @@ You can pass updates to the configuration through the command line like so:
 
 To extract alignments from the model run `dfaligner extract-alignments config/everyvoice-aligner.yaml --model path/to/model.ckpt`.
 
+To output alignments in text\_grid, run `dfaligner extract-alignments config/everyvoice-aligner.yaml --model path/to/model.ckpt --tg utt_count`, where `utt_count` is the number of utterances in your dataset.
+
+The extracted alignment can be found in `save_dir` specifified in `config/everyvoice-shared-data.yaml`
```
Comment (roedoejet) [inline README.md]:
```suggestion
The extracted alignment can be found in `save_dir` specified in `config/everyvoice-shared-data.yaml`
```

File: README.md [hunk 2]
```diff
@@ -82,6 +82,10 @@ You can pass updates to the configuration through the command line like so:
 
 To extract alignments from the model run `dfaligner extract-alignments config/everyvoice-aligner.yaml --model path/to/model.ckpt`.
 
+To output alignments in text\_grid, run `dfaligner extract-alignments config/everyvoice-aligner.yaml --model path/to/model.ckpt --tg utt_count`, where `utt_count` is the number of utterances in your dataset.
```
Comment (roedoejet) [inline README.md]:
```suggestion
To visualize some of your alignments and inspect them with [Praat](https://www.fon.hum.uva.nl/praat/), run `dfaligner extract-alignments config/everyvoice-aligner.yaml --model path/to/model.ckpt --tg utt_count`, where `utt_count` is the number of sample alignments you want to generate TextGrids for.
To output alignments in text\_grid format for your entire dataset, run `dfaligner extract-alignments config/everyvoice-aligner.yaml --model path/to/model.ckpt --tg utt_count`, where `utt_count` is the number of utterances in your dataset.
```

Comment (roedoejet) [reply README.md]: This revision might make it more clear why you would want to generate TextGrids, and also gives some motivation for why you might want to have `utt_count` be lower than the total number of utterances in the dataset.

**Review by roedoejet — CHANGES_REQUESTED** [review]
Thanks @MENGZHEGENG ! I just added a few suggested revisions. let me know what you think.

File: README.md (hunk 1 above)
Comment (MENGZHEGENG) [reply README.md]: Sorry for the typo!

File: README.md (hunk 2 above)
Comment (MENGZHEGENG) [reply README.md]: Yes you're definitely correct!  The revision is much more clearer! Thanks Aidan!

**Comment by joanise** [comment]
Just a small request: please squash all these commits into a single one with a conventional commit message briefly explaining the change, something like `docs: update README.md to explain how to produce TextGrids`. You can do that before merging, or using the "Squash and Merge" option here, both are fine.

**Comment by MENGZHEGENG** [comment]
Thank you so much Eric for the suggestions! Yes, the commits should be squashed when the PR is finally fixed.

**Review by roedoejet — APPROVED** [review]
Looks great - thanks @MENGZHEGENG !

**Pull request merged**